### PR TITLE
[Localization] Add Persian translations

### PR DESCRIPTION
**Pull request by behradjafari1**

### CODE REVIEW

**Comment by wstyres** [comment]
Thanks! Could you answer the following questions so that you can be properly credited?

**Are you a member of our discord? If so, what is your username so we can give you a special role to be notified of future translation updates**

**How would you like to be credited? (Name, Twitter Link, Website, etc.)**

**Comment by behradjafari1** [comment]
hi wilson
i guess something goes wrong!
i couldn't pull request it on master for unknown reason and i make a zebra-branch and send the pull request and now the file in your source code is english localization how can i send you the right persian one? im a regular github user tnx

**Comment by wstyres** [comment]
Everything is ok, I added the English versions to master so that they can be merged properly. There is nothing else you need to do unless you'd like to be credited and answer the questions I posted above. I will merge once the build passes.

**Comment by behradjafari1** [comment]
im just make a discord account so what should i do? and you can use my twitter : @behrad_bj

**Comment by wstyres** [comment]
Send me your username and join our Discord https://discord.gg/6CPtHBU

**Comment by behradjafari1** [comment]
my username is : behradjafari

File: Zebra/fa.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,281 +1,266 @@
...
-"Error" = "Error";
-"Retry" = "Retry";
-"Edit" = "Edit";
-"Login" = "Login";
-"Close" = "Close";
-"Done" = "Done";
+"Cancel" = "لغو";
+"No" = "خیر";
+"Ok" = "باشه";
+"Error" = "ارور";
```
Comment (Ali-Siya) [inline Zebra/fa.lproj/Localizable.strings]: خطا

File: Zebra/fa.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,281 +1,266 @@
...
+"Retry" = "دوباره امتحان کن";
```
Comment (Ali-Siya) [inline Zebra/fa.lproj/Localizable.strings]: سعی مجدد

File: Zebra/fa.lproj/Localizable.strings [hunk 3]
```diff
@@ -1,281 +1,266 @@
...
+"Edit" = "اصلاح";
```
Comment (Ali-Siya) [inline Zebra/fa.lproj/Localizable.strings]: ویرایش

File: Zebra/fa.lproj/Localizable.strings [hunk 4]
```diff
@@ -1,281 +1,266 @@
...
-"Wish List" = "Wish List";
-"No items in Wish List" = "No items in Wish List";
-"Community" = "Community";
-"Join our Discord" = "Join our Discord";
-"Follow me on Twitter" = "Follow me on Twitter";
+"Home" = "خانه";
+"Info" = "مشخصات";
+"Welcome to the Zebra Beta!" = "به زبرا نسخه ی بتا خوش آمدید";
+"Changelog" = "لیست تغییرات";
+"Community Repos" = "مخزن های معتبر";
```
Comment (Ali-Siya) [inline Zebra/fa.lproj/Localizable.strings]: منابع جامعه جیلبرک

File: Zebra/fa.lproj/Localizable.strings [hunk 5]
```diff
@@ -1,281 +1,266 @@
...
 
 //Credits
 
-"Credits" = "Credits";
-"Creators" = "Creators";
-"Contributors" = "Contributors";
-"Translators" = "Translators";
-"Libraries" = "Libraries";
+"Credits" = "اعتبار نامه ها";
+"Creators" = "تهیه کنندگان";
```
Comment (Ali-Siya) [inline Zebra/fa.lproj/Localizable.strings]: سازندگان

File: Zebra/fa.lproj/Localizable.strings [hunk 6]
```diff
@@ -1,281 +1,266 @@
...
-"OLED" = "OLED";
+"Settings" = "تنظیمات";
+"Version" = "نسخه";
+"Graphics" = "گرافیک ها";
+"Featured" = "ویژه";
+"News" = "اخبار";
+"Miscellaneous" = "متفرقه";
+"Advanced" = "تنظیمات پیشرفته";
+
+"Report a Bug" = "گزارش باگ نرم افزاری";
```
Comment (Ali-Siya) [inline Zebra/fa.lproj/Localizable.strings]: گزارش خرابی

**Pull request merged**